### ISSUE TITLE
Needs fiddling if RPC support is not in glibc

### ISSUE THREAD

**Issue by akhuettel** [lab-measurement/Lab-VXI11#1]
glibc is about to drop RPC support (it's marked obsolete and not built by default in glibc-2.26). This makes the build fail:
```
x86_64-pc-linux-gnu-gcc -c  -I. -Wall -O2 -pipe -march=native -Wall   -DVERSION=\"0.02\" -DXS_VERSION=\"0.02\" -fPIC "-I/usr/lib64/perl5/5.26.1/x86_64-linux/CORE"   VXI11.c
In file included from VXI11.xs:8:0:
./vxi11.h:9:21: fatal error: rpc/rpc.h: No such file or directory
 #include <rpc/rpc.h>
```
The replacement is libtirpc; the package needs to add /usr/include/libtirpc to the include path and link against -ltirpc ...

See e.g. 
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=9d0bb963bb94305320ae676636c206cb4625c8cb

**Comment by amba** [comment]
Think I will call pkg-config in the Makefile.PL to find libtirpc.
Like I do it in USB::LibUSB with libusb.

**Comment by amba** [comment]
Fixed in version 0.03. Just uploaded it to CPAN.
